sockeye/inference.py

```diff
@@ -551,9 +551,13 @@ def _make_result(self,
         :param attention_matrix: Attention matrix.
         :return: TranslatorOutput.
         """
+        # remove special sentence start symbol (<s>) from the output:
+        target_ids = target_ids[1:]
+        attention_matrix = attention_matrix[1:,:]
```

Comment:
why this? have you checked that this is compatible with the other output handlers?

Reply:
no strong feelings, but I thought it was more consistent to have the target tokens match the target string. In the attention matrix we anyway don't have any entries for the beginning of sentence symbol so it doesn't add any value. I checked with the alignment plotting output handler. As the attention_matrix is consistent with the source and target tokens I don't expect this to break any of the handlers.

Reply:
makes sense.